plugin.go

```diff
@@ -0,0 +1,167 @@
+package main
```

Comment:
gofmt this file

Reply:
grr I always forget to gofmt. Sorry about that and done :)